default/stringtables/en.txt

```diff
@@ -7935,57 +7943,110 @@ Launch Bay
 
 PC_FIGHTER_BAY_DESC
 '''Launch bays are the access point for [[encyclopedia FIGHTER_TECHS]] to enter or exit a ship.
-Each launch bay allows a number of fighters to launch each combat round.
-'''
+Each launch bay allows a number of fighters to launch each combat round.  Launched fighters take one combat round to reach firing range.'''
 
 # Ship part classes
 PC_SHIELD
 Shields
 
+PC_SHIELD_DESC
+'''Shield parts reduce the [[encyclopedia DAMAGE_TITLE]] sustained from each hit by the strength rating of the shield part. Only the remainder of a weapons damage value will affect the ship. Not all sources of damage will be reduced by a shield, for example [[encyclopedia FIGHTER_TECHS]] are unaffected by ship shields.
```

Comment:
"... from direct fire weapon hit by the shield part's strength."

lose the "Only ... "sentence.

Just say "Fighter damage is unaffected by shields."